gossip/src/cluster_info.rs

```diff
@@ -1207,7 +1207,7 @@ impl ClusterInfo {
         let mut gossip_crds = self.gossip.crds.write().unwrap();
         for entry in entries {
             if let Err(err) = gossip_crds.insert(entry, timestamp(), GossipRoute::LocalMessage) {
-                error!("Insert self failed: {err:?}");
+                warn!("Insert self failed: {err:?}");
```

Comment:
CC @gregcusack . I believe the expectation is this goes away when we remove legacy contact info or something

Reply:
sorry completely missed this. yes this will go away when we remove `NodeInstance` (which is currently deprecated). behzad and i decided to leave it since it doesn't cause any issues but ya should be a warning